monitor/census.go

```diff
@@ -825,11 +834,17 @@ func SegmentUploaded(nonce, seqNo uint64, uploadDur time.Duration) {
 }
 
 func (cen *censusMetricsCounter) segmentUploaded(nonce, seqNo uint64, uploadDur time.Duration) {
-	cen.lock.Lock()
-	defer cen.lock.Unlock()
 	stats.Record(cen.ctx, cen.mSegmentUploaded.M(1), cen.mUploadTime.M(float64(uploadDur/time.Second)))
 }
 
+func AuthWebhookFinished(dur time.Duration) {
+	census.authWebhookFinished(dur)
+}
+
+func (cen *censusMetricsCounter) authWebhookFinished(dur time.Duration) {
+	stats.Record(cen.ctx, cen.mAuthWebhookTime.M(dur.Seconds()))
```

Comment:
Is there an advantage to using seconds as opposed to milliseconds? Theoretically *most* of these values should be in the millisecond range, which might be nicer with fewer non-floaty numbers...